src/nvim/eval.c

```diff
@@ -12404,11 +12404,10 @@ static void f_jobstop(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 
   if (argvars[0].v_type != VAR_NUMBER) {
     // Only argument is the job id
-    EMSG(_(e_invarg));
     return;
```

Comment:
why was this removed? If an invalid argument type is provided, it's not useful to just ignore it.